RecoJets/Configuration/python/RecoPFJets_cff.py

```diff
@@ -105,3 +105,23 @@
                            ak4PFJetsSK
     )
 recoPFJetsWithSubstructure=cms.Sequence(recoPFJetsWithSubstructureTask)
+
+from RecoHI.HiJetAlgos.HiRecoPFJets_cff import PFTowers, akPu3PFJets, akPu4PFJets, kt4PFJetsForRho, hiFJRhoProducer, akCs4PFJets
+
+recoPFJetsHITask =cms.Task(fixedGridRhoAll,
+                           fixedGridRhoFastjetAll,
+                           fixedGridRhoFastjetCentral,
+                           fixedGridRhoFastjetCentralChargedPileUp,
+                           fixedGridRhoFastjetCentralNeutral,
+                           ak4PFJets,
+                           ak4PFJetsCHS,
+                           ak8PFJetsCHS,
```

Comment:
CHS requires pfNoPileUpJMETask.
Please add it to this task

Reply:
I think that this should resolve what was brought up in https://github.com/cms-sw/cmssw/issues/24587#issuecomment-422481794

Reply:
@stepobr @mandrenguyen 
do we have a fix to this already?

Reply:
@slava77 I'm testing the fix right now, will soon make a pull request.